tests/unit/s2n_x509_validator_test.c

```diff
@@ -1405,12 +1405,12 @@ int main(int argc, char **argv) {
 
         struct s2n_pkey public_key_out;
         EXPECT_SUCCESS(s2n_pkey_zero_init(&public_key_out));
-        s2n_pkey_type pkey_type;
+        s2n_pkey_type pkey_type = S2N_PKEY_TYPE_UNKNOWN;
         EXPECT_ERROR_WITH_ERRNO(s2n_x509_validator_validate_cert_chain(&validator, connection, chain_data, chain_len,
                 &pkey_type, &public_key_out), S2N_ERR_CERT_UNTRUSTED);
 
         s2n_stuffer_free(&chain_stuffer);
-        EXPECT_EQUAL(S2N_PKEY_TYPE_RSA, pkey_type);
+        EXPECT_TRUE(S2N_PKEY_TYPE_UNKNOWN == pkey_type);
```

Comment:
Nit: EXPECT_EQUAL, not EXPECT_TRUE. I'm not sure why just this one changed lol.

Reply:
Sorry, I'm not sure either. - fixed: fd87c6c2f453e53dce0087370f6bd0ad2511e1f1